projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -276,11 +292,13 @@ private function prepare_menu_item( array $menu_item ) {
 	/**
 	 * Sets up a submenu item for consumption by Calypso.
 	 *
-	 * @param array $submenu_item Submenu item.
-	 * @param array $menu_item    Menu item.
+	 * @param array $submenu_item    Submenu item.
+	 * @param array $menu_item       Menu item.
+	 * @param bool  $admin_is_parent Check if the parent menu is admin.php.
+	 *
 	 * @return array Prepared submenu item.
 	 */
-	private function prepare_submenu_item( array $submenu_item, array $menu_item ) {
+	private function prepare_submenu_item( array $submenu_item, array $menu_item, $admin_is_parent ) {
```

Comment:
If we can't move the above code block into `prepare_menu_item_url()` and `$admin_is_parent` stays around as a parameter, let's also give it a default value, just in case.